charts/opentelemetry-operator/values.yaml

```diff
@@ -121,9 +121,13 @@ manager:
     # Create default rules for monitoring the manager
     defaultRules:
       enabled: false
-    # additional labels on the PrometheusRule
+      ## Additional labels for PrometheusRule alerts
+      additionalRuleLabels: {}
+      ## Additional annotations for PrometheusRule alerts
+      additionalRuleAnnotations: {}
+    # additional labels on the PrometheusRule object
```

Comment:
Can this be handled via `prometheusRule.groups` instead?

Reply:
`prometheusRule.groups` allows defining new custom alerts on which we can already add any labels/annotations.
This for the [existing](https://github.com/open-telemetry/opentelemetry-helm-charts/blob/4d3d0548f409b2da6d737228f1f67a3f5602b21d/charts/opentelemetry-operator/templates/prometheusrule.yaml#L23) alerts included with the chart.

Reply:
I'm wondering if these rules should follow the same rules as presets in the collector chart or if they should be configurable like the defaults in the demo chart.

In theory you could disable the default rules and recreate the exact rules, including labels, via the groups field right?

Reply:
> In theory you could disable the default rules and recreate the exact rules, including labels, via the groups field right?

Absolutely, but then you would need to manage them separately from the chart ... which is undesirable for us.

Reply:
What do you mean by manage them separately from the chart?

Reply:
I meant keeping them into the values an not using the default templates would require some work on our side vs being able to label the existing templated alerts.
This may cause divergence from the main chart in the future as more alerts are being added/changed.
I see the same setup in [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml#L97C3-L97C23)
Any concerns for merging this ?

Reply:
@TylerHelmuth can we merge this ?